### ISSUE TITLE
Auto-export stuck on "pending"

### ISSUE THREAD

**Issue by CarbonFixer** [ZotPlus/zotero-better-bibtex#426]
Report ID: 9NQF7HP6

It used to work fine and I don't know what changed.
When adding a new entry to my collection, the defined .bib file does not update. If I go to Preferences -> Better BibTeX I see that (1) Automatic Export On Change is marked and (2) the status of my collection is "pending". if I click on the "Export" button on the bottom it updates instantly, but I have to do it again for each entry I add. The automatic export always gets stuck on "pending". 
Any ideas on what could have gone wrong?

**Comment by retorquere** [comment]
It looks like BBT did not initialize properly. There's at least one problem I found based on your error report, but there must be others. Could you enable extended debugging in BBT, install https://s3.eu-central-1.amazonaws.com/zotplus-builds/zotero-better-bibtex-1.6.25-travis-2339.xpi, and submit a new error report?

**Comment by CarbonFixer** [comment]
After looking at the instructions for use again, I see you are not supposed to run Zotero standalone in parallel to the Firefox extension, which I did the day I posted the bug report. I can't reproduce the issue now though.

**Comment by retorquere** [comment]
Yeah, that would do it. Not sure how I'm going to tackle that yet.

**Comment by arvidj** [comment]
Hello, just popping in to say that I have the same issue (stuck on pending). I use the stand-alone Zotero with the Chrome add-on. I'm using the last release (1.6.45). Not sure whether I should open a new issue?

**Comment by retorquere** [comment]
Please do open a new issue.
